charts/lotus-stats/templates/deployment.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "lotus-stats.fullname" . }}
+  labels:
+    {{- include "lotus-stats.labels" . | nindent 4 }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      {{- include "lotus-stats.labels" . | nindent 6 }}
+  template:
+    metadata:
+      labels:
+        {{- include "lotus-stats.labels" . | nindent 8 }}
+    spec:
+      serviceAccountName: {{ .Values.serviceAccountName }}
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      containers:
+      - name: stats
+        {{- with .Values.securityContext }}
+        securityContext:
+          {{- toYaml . | nindent 8 }}
+        {{- end }}
+        image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+        imagePullPolicy: {{ .Values.image.pullPolicy }}
+        command: ["/usr/local/bin/lotus-stats", "run"]
+        args:
+          - -no-sync={{ .Values.nosync }}
+          - -head-lag={{ .Values.headlag }}
+          - -height={{ .Values.height }}
+        env:
+          - name: FULLNODE_API_INFO
+            value: {{ .Values.lotusApiInfo }}
```

Comment:
does this contain the jwt token?

Reply:
No, the api is by default read-only and this service does not required additional privs above read.